pandas/tests/extension/list/test_list.py

```diff
@@ -60,6 +62,28 @@ def data_missing(dtype):
     return arr
 
 
+@pytest.fixture
+def data_for_sorting(data_for_grouping):
+    """
+    Length-3 array with a known sort order.
+
+    This should be three items [B, C, A] with
+    A < B < C
+    """
+    pytest.skip("ListArray does not support sorting")
+
+
+@pytest.fixture
+def data_missing_for_sorting(data_for_grouping):
+    """
+    Length-3 array with a known sort order.
+
+    This should be three items [B, NA, A] with
+    A < B and NA missing.
+    """
+    pytest.skip("ListArray does not support sorting")
```

Comment:
why not?  dont python lists have order semantics?

Reply:
Pyarrow doesn't directly support sorting lists, and if it did I'm not sure it would use the same semantics as Python (I don't know how other languages would care for that)

I suppose we could implement something ourselves to cast all elements to Python and use the language's list semantics, but that's likely a big change better left to a follow up

Reply:
OK